src/frontend/Navigation/ScreenGroups/TabBar/TabBarIcon.tsx

```diff
@@ -0,0 +1,19 @@
+import React, {FC} from 'react';
+import MaterialIcons from 'react-native-vector-icons/MaterialIcons';
+import {TabBarIconProps} from '../../types';
+import {COMAPEO_BLUE, MEDIUM_GREY} from '../../../lib/styles';
+
+export interface TabBarIcon extends TabBarIconProps {
+  isFocused: boolean;
+  iconName: string;
+}
+
+export const TabBarIcon: FC<TabBarIcon> = ({size, iconName, isFocused}) => {
+  return (
+    <MaterialIcons
+      name={iconName}
+      size={size}
+      color={isFocused ? COMAPEO_BLUE : MEDIUM_GREY}
+    />
+  );
+};
```

Comment:
think the `isFocused` prop isn't really necessary when `focused` is part of the built-in props

```suggestion
export interface TabBarIcon extends TabBarIconProps {
  iconName: string;
}

export const TabBarIcon: FC<TabBarIcon> = ({size, iconName, focused}) => {
  return (
    <MaterialIcons
      name={iconName}
      size={size}
      color={focused ? COMAPEO_BLUE : MEDIUM_GREY}
    />
  );
};
```

Reply:
Yeah, I agree with you, thanks ;)